aws/app/outputs.tf

```
@@ -17,3 +17,33 @@ output "ecs_service_name" {
   description = "ECS service name"
   value       = aws_ecs_service.form_viewer.name
 }
+
+output "ecs_iam_forms_secrets_manager_policy_arn" {
+  description = "IAM policy for access to Secrets Manager"
+  value       = aws_iam_role.forms_secrets_manager.arn
```

This just needs to be the policy:
```suggestion
  value       = aws_iam_policy.forms_secrets_manager.arn
```